Index: src/Actions/DrawCard.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Actions;\r\n\r\npublic class DrawCard {\r\n    // take a card of the pile and thus removes one card from the pile (pile -1)\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Actions/DrawCard.java b/src/Actions/DrawCard.java
--- a/src/Actions/DrawCard.java	(revision cc775cd4accfb1c69b5cb511d135663405a8a75b)
+++ b/src/Actions/DrawCard.java	(date 1701968066641)
@@ -2,4 +2,5 @@
 
 public class DrawCard {
     // take a card of the pile and thus removes one card from the pile (pile -1)
+    //testtest123
 }
